src/controllers/profile.js

```diff
@@ -0,0 +1,10 @@
+const Product = require('../models/product');
+const ErrorResponse = require('../utils/errorResponse');
+
+exports.getMyProductsForSale = async (req, res, next) => {
```

Comment:
you could give this a better name, like `getUserProducts`

Reply:
changed name of `getMyProductsForSale` to `getUserProducts`.